src/main/java/keycontacts/model/student/GradeLevel.java

```diff
@@ -0,0 +1,119 @@
+package keycontacts.model.student;
+
+import java.util.regex.PatternSyntaxException;
+
+/**
+ * Represents a Student's grade level field in the student object.
+ */
+public class GradeLevel {
+
+    public static final String MESSAGE_CONSTRAINTS = "Grade level should only contain alphanumeric characters "
+            + "and spaces, exam board (e.g. ABRSM, Trinity) "
+            + "and grade level should be separated with a space. Grade level should not exceed 50 characters, "
```

Comment:
Minor nit, but it should be `Exam board should not exceed...`. Might need to check if this problem exists elsewhere